pkg/apis/kops/componentconfig.go

```diff
@@ -184,10 +184,10 @@ type KubeAPIServerConfig struct {
 	CloudProvider string `json:"cloudProvider,omitempty" flag:"cloud-provider"`
 	// SecurePort is the port the kube runs on
 	SecurePort int32 `json:"securePort,omitempty" flag:"secure-port"`
-	// InsecurePort is the port the insecure api runs
-	InsecurePort int32 `json:"insecurePort,omitempty" flag:"insecure-port"`
-	// Address is the binding address for the kube api
-	Address string `json:"address,omitempty" flag:"address"`
+	// Deprecated: InsecurePort is the port the insecure api runs
+	InsecurePort int32 `json:"insecurePort,omitempty" flag:"insecure-port" flag-empty:"-1"`
```

Comment:
Does 0 work as the flag-empty value?

Reply:
First step, I want `0` to disable insecure http schema. So for <v1.10 it continue support `0`.